app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.kt

```diff
@@ -108,7 +108,7 @@ class DescriptionEditFragment : Fragment() {
         val type = if (pageTitle.description == null) DescriptionEditFunnel.Type.NEW else DescriptionEditFunnel.Type.EXISTING
         funnel = DescriptionEditFunnel(WikipediaApp.getInstance(), pageTitle, type, invokeSource)
         funnel.logStart()
-        EditAttemptStepEvent.logInit()
+        EditAttemptStepEvent.logInit(pageTitle.wikiSite.languageCode)
```

Comment:
Would it be possible to initialize the `EditAttemptStepEvent` with the `WikiSite` on the top so we don't have to pass `languageCode` every time when we call it.